packages/react-router-dom-v5-compat/lib/components.tsx

```diff
@@ -86,7 +86,7 @@ export function StaticRouter({
     pathname: locationProp.pathname || "/",
     search: locationProp.search || "",
     hash: locationProp.hash || "",
-    state: locationProp.state || null,
+    state: locationProp.state ?? null,
```

Comment:
While the nullish coalescing operator is the "proper" way to do this, we have a lint rule that doesn't allow it because when it gets transpiled away it results in a fairly large bundle bloat so we stopped using it in the code until a major version bump when we can update our tranpsilation settings and minimum browser support.  Would you mind changing this to:

```suggestion
    state: locationProp.state != null ? locationProp.state : null,
```

Reply:
Cool!!! I'll apply it!!